### ISSUE TITLE
Glow/Username Issue

### ISSUE THREAD

**Issue by pabjr17** [Zrips/CMI#4638]
**Description of issue:**

---
The following bug report was brought to us today, done with cmi's /glow command, when using a color with /glow it changes the color of the nametag, and when turning glow off it does not seem to remove that color. I am unsure If I am missing a config or not.
Our bug report;

**Please write a brief description of the bug or issue.**
When getting out /glow your name stays whatever color your /glow was

**Please describe in detail the bug or issue.**
Basically when coming out of /glow it was noticed that the name stays whatever color your glow was

**Evidence:**
https://imgur.com/ud95PTW - Myself
https://imgur.com/L3LbM88 - Myself
https://imgur.com/4pdc9UD - Myself (out of /glow red)
https://imgur.com/U8YidSQ - Myself
https://imgur.com/pAOpgkY - 

---

**Cmi Version (using`/cmi version`): 8.7.10.4** 

**Server Type (Spigot/Paperspigot/etc): Paper** 

**Server Version (using `/ver`): https://i.imgur.com/TlwAJBb.png**

**Comment by Zrips** [comment]
Should be resolved with next update
